src/components/WelcomeModal/WelcomeModal.stories.tsx

```diff
@@ -15,42 +15,50 @@ export default {
   },
 };
 
-export const NewSpruceUser = () => (
-  <WelcomeModal
-    title="Welcome to the New Evergreen UI!"
-    param="hasUsedSpruceBefore"
-    carouselCards={newSpruceUser}
-  />
-);
+export const NewSpruceUser: StoryObj<typeof WelcomeModal> = {
```

Comment:
Think you need an `MockedProvider` in this file as well!

<img width="1091" alt="Screenshot 2023-03-22 at 7 41 25 PM" src="https://user-images.githubusercontent.com/47064971/227062893-33f844b6-07bd-4860-93bf-42cf5966e5f9.png">